pkg/server/plugin/upstreamca/spire/spire_test.go

```diff
@@ -240,20 +241,13 @@ func TestSpirePlugin_SubmitValidCSR(t *testing.T) {
 	m, done := newWithDefault(t, server.napiServer.addr, server.wapiServer.socketPath)
 	defer done()
 
-	const testDataDir = "_test_data/csr_valid"
-	validCsrFiles, err := ioutil.ReadDir(testDataDir)
+	validSpiffeID := "spiffe://localhost"
+	csr, err := util.NewCSRTemplate(validSpiffeID)
 	require.NoError(t, err)
 
-	for _, validCsrFile := range validCsrFiles {
-		csrPEM, err := ioutil.ReadFile(filepath.Join(testDataDir, validCsrFile.Name()))
-		require.NoError(t, err)
-		block, rest := pem.Decode(csrPEM)
-		require.Len(t, rest, 0)
-
-		resp, err := m.SubmitCSR(ctx, &upstreamca.SubmitCSRRequest{Csr: block.Bytes})
-		require.NoError(t, err)
-		require.NotNil(t, resp)
-	}
+	resp, err := m.SubmitCSR(ctx, &upstreamca.SubmitCSRRequest{Csr: csr})
+	require.NoError(t, err)
+	require.NotNil(t, resp)
 }
 
 func TestSpirePlugin_SubmitInvalidCSR(t *testing.T) {
```

Comment:
Please add a test case with an invalid sequence of bytes.

Reply:
Same as above.